libs/oeth/history/src/hooks.ts

```diff
@@ -202,3 +207,53 @@ export const useAggregatedHistory = (
     },
   );
 };
+
+export const useWOETHHistory = () => {
```

Comment:
wrapping this in a custom useQuery would give many advantages

Reply:
but would it really right now? I just don't see any immediate advantage or reason to do it

Reply:
yes, multiple advantages:
- colocate all logic for creating this data model into this one hook/selector
- running queries in parrallel instead of sequential when possible
- caching and memoization at query and selector levels
- easy access to query state and controls (isFetching, isLoading, refetch,...)
- easy invalidation
- consistent api with the rest of the dApp

Reply:
I guess the real winner for me is in merging the query state/controls.
I went ahead and did this and I like the way it looks. I also like that the complicated hook logic is gone.